### ISSUE TITLE
"Please wait executing macro" sign showing on top of dialog boxes of the macro

### ISSUE THREAD

**Issue by bernatagulloesbrina** [TabularEditor/TabularEditor3#622]
Hello, 

in the lastest version 3.3.3 I see that when I run a macro with user interaction to select tables or measures, the "Please wait executing macro" sign shows on top which might confuse users. 

Is there a way to turn it off somehow? 

![image](https://user-images.githubusercontent.com/40455069/183237599-5a6242f3-588f-49aa-948f-8575f8305db4.png)

**Comment by bernatagulloesbrina** [comment]
this is the macro I am executing if it helps with debug
https://github.com/bernatagulloesbrina/TabularEditor-Scripts/blob/main/Advanced/One-Click%20Macros/Filtered%20Measures%20by%20column%20values.csx

**Comment by otykier** [comment]
That’s a bug, thanks for reporting.

**Comment by otykier** [comment]
Fixed in [3.3.5](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.3.5). Thanks for reporting!

**Comment by bernatagulloesbrina** [comment]
Hello, 

this issue is somehow persisting in 3.3.5 

![image](https://user-images.githubusercontent.com/40455069/195067083-47a123d7-9478-451e-ab01-fd645800d19c.png)

**Comment by bernatagulloesbrina** [comment]
Hello, on versión 3.4 is still there too

![imatge](https://user-images.githubusercontent.com/40455069/212529154-cd84b678-3f8c-4b95-b3e6-ee3bab3ad94d.png)

**Comment by mlonsk** [comment]
Thank you for updating the issue, we will look into it.

**Comment by otykier** [comment]
If you're using custom UI, you can disable the "Please wait" message by including this line at the top of your script:

```csharp
WaitFormVisible = false;
```

This code works in TE2 as well, but it does nothing (as TE2 does not display a wait form).

The built-in methods to display information on the UI (`Info`,  `Output`, `SelectObject`, `SelectMeasure`, etc.), automatically hide the wait form temporarily.

**Comment by otykier** [comment]
**Update:** Note that if you plan to use this is in a TE2 script, or if you want to save as a macro, you'll have to change the code slightly to:

```csharp
ScriptHelper.WaitFormVisible = false;
```

**Comment by mlonsk** [comment]
Hi @bernatagulloesbrina 
This issue has been resolved in Tabular Editor 3.11.0 - sorry for not updating sooner.
